openassessment/xblock/staff_area_mixin.py

```diff
@@ -343,6 +343,10 @@ def get_student_submission_context(self, student_username, submission):
             'student_username': student_username,
         }
 
+        if submission['file_url'] is not None:
+            context["staff_file_url"] = submission["file_url"]
```

Comment:
Could this logic be inside the ``try`` block as it wouldn't then need the check? Could there be times when the submission comes in with the ``file_url`` already populated?